src/main/java/uk/gov/hmcts/reform/hmc/data/HearingRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.hmc.data;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional(propagation = Propagation.REQUIRES_NEW)
+@Repository
+public interface HearingRepository extends CrudRepository<HearingEntity, Long> {
+
+    @Query("select h from HearingEntity h where h.id = :id")
+    HearingEntity findHearing(Long id);
```

Comment:
```suggestion
```
We can delete this as the method is already provided (will add other comment in invoking class)